aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/plugins/latest_entries.html

```diff
@@ -1,12 +1,18 @@
 {% load i18n %}
 
 <div class="plugin plugin-jobs">
-    <p>{% blocktrans with total=instance.job_offers.count %}Showing {{ total }} Job Offers{% endblocktrans %}
-	<ul class="jobs-latest-entries">
-	{% for job in instance.job_offers %}
-		<li><p><a href="{{ job.get_absolute_url }}">{{ job }}</a></p></li>
-	{% empty %}
-		<li class="jobs-empty"><p>{% trans "There are currently no job openings." %}</p></li>
-	{% endfor %}
-	</ul>
+{% if plugin_configuration_error %}
```

Comment:
@cyberstar be aware that this changed in: https://github.com/aldryn/aldryn-jobs/pull/61